components/ws-manager/pkg/manager/manager.go

```diff
@@ -1511,7 +1497,7 @@ func newWssyncConnectionFactory(managerConfig config.Configuration) (grpcpool.Fa
 	return func(host string) (*grpc.ClientConn, error) {
 		var (
 			addr           = fmt.Sprintf("%s:%d", host, port)
-			conctx, cancel = context.WithTimeout(context.Background(), wsdaemonDialTimeout)
+			conctx, cancel = context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
seems like we removed const value and replaced it with hard coded value? :thinking: why?

Reply:
Changed the place of the definition to be in the same file and near the only place where it is used